nginx.conf

```diff
@@ -27,11 +27,7 @@ events {
 
 http {
     include nginx.conf.d/mime.types;
-    lua_shared_dict auto_ssl 1m;
```

Comment:
These are not needed since we aren't using auto-ssl anymore. 

The two below were moved to ssl-shared.conf so they're with all the other ssl settings.